examples/README.md

```diff
@@ -8,14 +8,17 @@ Naming convention: `data_method` or `data_method_lightn` for lightning (optional
 
 All data in examples are from the public domain so they will be downloaded into local directories, either automatically or following instructions in specific examples. To keep this repository size small, we do not upload data here unless the size is less than 300KB. This is done by setting `.gitignore`. Data can be shared at [PyKale Data Repository](https://github.com/pykale/data) or other external locations, e.g., Google Drive.
 
-## Examples available in three areas
+## Examples are available on synthetic data and three application areas
 
-* Image/video recognition
-  * Image classification on [CIFAR via transformers](https://github.com/pykale/pykale/tree/master/examples/cifar_cnntransformer), [CIFAR via ISONet](https://github.com/pykale/pykale/tree/master/examples/cifar_isonet), [digits via domain adaptation](https://github.com/pykale/pykale/tree/master/examples/digits_dann_lightn)
-  * [Action recognition via domain adaptation](https://github.com/pykale/pykale/tree/master/examples/action_dann_lightn)
-    * [Video loading](https://github.com/pykale/pykale/tree/master/examples/video_loading)
-* Bioinformatics/graph analysis
-  * [Drug-target interaction prediction](https://github.com/pykale/pykale/tree/master/examples/bindingdb_deepdta)
-  * [Polypharmacy side effect prediction](https://github.com/pykale/pykale/tree/master/examples/drug_gripnet)
-* Medical image analysis
-  * [Cardiac MRI diagnosis](https://github.com/pykale/pykale/tree/master/examples/cmri_mpca)
+- Synthetic data
+  - [Toy data classification with domain adaptation](https://github.com/pykale/pykale/tree/main/examples/toy_domain_adaptation)
+- Image/video recognition
+  - Image classification on [CIFAR via transformers](https://github.com/pykale/pykale/tree/master/examples/cifar_cnntransformer), [CIFAR via ISONet](https://github.com/pykale/pykale/tree/master/examples/cifar_isonet), [digits via domain adaptation](https://github.com/pykale/pykale/tree/master/examples/digits_dann_lightn), [digits/office via multi-source domain adaptation](https://github.com/pykale/pykale/tree/main/examples/multisource_adapt)
+  - [Action recognition via domain adaptation](https://github.com/pykale/pykale/tree/master/examples/action_dann_lightn)
+  - [Video loading](https://github.com/pykale/pykale/tree/master/examples/video_loading)
+- Bioinformatics/graph analysis
```

Comment:
"Graph" is a term of methodology but not application for me. How about "Graph analysis for bioinformatic data" or "Bioinformatic data analysis with graph neural networks"?